osidb/models.py

```diff
@@ -2450,7 +2450,21 @@ def save(self, *args, bz_api_key=None, jira_token=None, **kwargs):
             and self.type == self.TrackerType.JIRA
         ):
             # sync to Jira
-            tracker_instance = TrackerSaver(self, jira_token=jira_token).save()
+
+            actual_jira_issuetype = "Vulnerability"
+            if JiraBugIssuetype.objects.filter(
+                project=PsModule.objects.get(
+                    name=self.affects.first().ps_module
+                ).bts_key
+            ).exists():
+                actual_jira_issuetype = "Bug"
```

Comment:
One thing to be careful about is that before the first PS constants collection after the release we are actually not going to default to the old behavior but to the new one. We could overcome it by checking whether there is actually at least some **JiraBugIssuetype** but that might that cause problems when all of them adapt and the field gets empty. Otherwise we could manually do the sync in console right after the release or do it eg. inside a migration.

Reply:
How would we make it behave sanely once the last project switches to Vulnerability issuetype? If we add a check here and now, then in that distant (?) point in future, it will trigger confusing and already-forgotten behavior of switching all new trackers back to Bug. Maybe I'm missing something.

Doing the first sync manually sounds good to me.

Reply:
Yeah, I am OK with it too. Just keep it in mind before we go to production and we should be ready to just hit a command and run it as I think for the PS constants last time it was quite laborious.